content/recipes/interoperability/registering-swisslipids-identifiers.md

```diff
@@ -0,0 +1,162 @@
+# Registering SwissLipids identifiers in Wikidata
+
+ ````{panels_fairplus}
+:identifier_text: FCBxxx
+:identifier_link: 'https://w3id.org/faircookbook/FCBxxx'
+:difficulty_level: 5
+:recipe_type: hands_on
+:reading_time_minutes: 15
+:intended_audience: bioinformatician, data_scientist, data_engineer
+:has_executable_code: nope
+:recipe_name: Registering SwissLipids identifiers in Wikidata
+```` 
+
+## Main Objectives
+
+The objective of this recipe is to describe how external identifiers can be added to Wikidata in an automated.
+
+___
+
+
+## Requirements
+
+To record identifiers of some resource to make the content more findable and more interoperable, first a [Wikidata](https://www.wikidata.org/) property
+needs to be requested. That process is described [here](https://www.wikidata.org/wiki/Wikidata:Property_proposal).
+
+---
+
+
+## Main Content
+
+Making datasets more findable and databases more interoperable, it is recommended to share not just top-level metadata, but also record-level
```

Comment:
a bit unclear as it leads to the question "what is top-level metadata and what is record-level metadata? ". I would suggest rephrasing or clarifying what you mean here.